src/gateway/routers/Aggregator.sol

```diff
@@ -31,9 +33,8 @@ contract Aggregator is Auth, IAggregator {
 
     address[] public routers;
     mapping(address router => Router) public validRouters;
```

Comment:
I know this is not the topic of this PR but I will just write down this suggestion. 
Can't remember where exactly I read about this ( code clean book or Martin Fowler or some where else ) but using `valid` and `invalid` is ambiguous. `Valid` doesn't show how this is valid. 
In our case probably a better name could be `approvedRouters` or `activeRouters` or `whitelistedRouters` .
This is just a side comment while I'm revising the code.

Reply:
Yeah I agree, will change it to `activeRouters`, I like that